addons/web_editor/static/lib/odoo-editor/test/spec/editor.test.js

```diff
@@ -1322,12 +1322,12 @@ X[]
                         contentAfter: '<p><b>ax[]</b></p>',
                     });
                     await testEditor(BasicEditor, {
-                        contentBefore: '<p><span>cd</span><span class="removeme" oe-zws-empty-inline="">[]\u200B</span></p>',
+                        contentBefore: '<p><span odoo-test-ignore-attr>cd</span><span class="removeme" oe-zws-empty-inline="">[]\u200B</span></p>',
```

Comment:
@dmo-odoo Is this another span that we should not ignore for the relevance of the test?

Reply:
I'm not sure this particular one serves any actual purpose, but in doubt I'd leave it.